apps/web/src/views/LiquidStaking/components/FAQs.tsx

```diff
@@ -89,8 +18,9 @@ export const LiquidStakingFAQs = () => {
         </Heading>
       </CardHeader>
       <CardBody>
-        {config(t).map(({ title, description }, i, { length }) => (
-          <FoldableText key={title} id={title} mb={i + 1 === length ? '' : '24px'} title={title}>
+        {config?.map(({ title, description }, i, { length }) => (
+          // eslint-disable-next-line react/no-array-index-key
+          <FoldableText key={i} mb={i + 1 === length ? '' : '24px'} title={title}>
```

Comment:
can we use title as key or add a manual id for the qa list instead of array index?

Reply:
done